comms.py

```diff
@@ -0,0 +1,81 @@
+from mcu import *
+
+class Communications:
+    
+    PWM_MIN = 1000
+    PWM_MID = 1500
+    PWM_MAX = 2000
+    
+    MOTOR_CLAW = 0 
+    MOTOR_FRONT = 1
+    MOTOR_RIGHT = 2
+    MOTOR_LEFT = 3 
+    MOTOR_BACK = 4
+    
+    INITIAL_PERCENT = 100
+    
+    def __init__(self, mcuVAR: MCUInterface, MULTIPLIER_PERCENT: int):
+        self.mcuVAR = mcuVAR
+        self.MULTIPLIER_PERCENT = MULTIPLIER_PERCENT
+
+    def read_send(key_pressed):
+        
+        self.mcuVAR.cmd_setMotorCalibration(MOTOR_CLAW,1000)
+        self.mcuVAR.cmd_setMotorCalibration(MOTOR_FRONT, 1000)
+        self.mcuVAR.cmd_setMotorCalibration(MOTOR_LEFT, 1000)
+        self.mcuVAR.cmd_setMotorCalibration(MOTOR_RIGHT, 1000)
+        self.mcuVAR.cmd_setMotorCalibration(MOTOR_BACK, 1000)
+
+        self.mcuVAR.cmd_setMotorMicroseconds(MOTOR_CLAW, PWM_MID)
+        self.mcuVAR.cmd_setMotorMicroseconds(MOTOR_FRONT, PWM_MID)
+        self.mcuVAR.cmd_setMotorMicroseconds(MOTOR_LEFT, PWM_MID)
+        self.mcuVAR.cmd_setMotorCalibration(MOTOR_RIGHT, PWM_MID)
+        self.mcuVAR.cmd_setMotorCalibration(MOTOR_BACK, PWM_MID)
+
+        
+        dict_motors = {
+            "e": [MOTOR_FRONT, MOTOR_BACK, OPPOSITE],
```

Comment:
I don't get the rationale behind making it opposite. I know that the thrusters are facing the same directions. If I set both thrusters to 100%, for example, I know that both thrusters individually would try to move forward. This applies to all with opposites as their index-2 value in the dictionary.